imports/server/bugsnag.ts

```diff
@@ -22,7 +22,12 @@ if (apiKey) {
     ],
     onError: (event) => {
       if (Fiber.current) {
-        const user = Meteor.user();
+        let user;
+        try {
+          user = Meteor.user();
```

Comment:
Should this be something like `await MeteorUsers.findOne(Meteor.userId()).fetchAsync()` since we're in `server` code?